src/Carnac.Logic/KeyPlayer.cs

```diff
@@ -3,7 +3,7 @@
 using System.Reactive.Linq;
 using Carnac.Logic.KeyMonitor;
 
-namespace Carnac.Tests
+namespace Carnac.Logic
```

Comment:
On closer inspection, it looks like `KeyStreamCapture` is just a test project to test the key capture..will move the file back